lang/reports/2016-03-21.md

```diff
@@ -0,0 +1,180 @@
+# Subteam report: lang 2016-03-21
+
+## Highlights
+
+We decided to accept two RFCs yesterday:
+
+- [RFC 1201](https://github.com/rust-lang/rfcs/pull/1201): Support
+  naked functions, which are functions without the standard prologue
+  and epilogue; this is primarily useful for low-level coding.
+- [RFC 1244](https://github.com/rust-lang/rfcs/pull/1244): Extend the
```

Comment:
I guess that should be [1422](https://github.com/rust-lang/rfcs/pull/1422), not 1244.